federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -68,35 +91,41 @@ type IngressController struct {
 
 	// Backoff manager for ingresses
 	ingressBackoff *flowcontrol.Backoff
+	// Backoff manager for configmaps
+	configMapBackoff *flowcontrol.Backoff
 
 	// For events
 	eventRecorder record.EventRecorder
 
 	ingressReviewDelay    time.Duration
+	configMapReviewDelay  time.Duration
 	clusterAvailableDelay time.Duration
 	smallDelay            time.Duration
 	updateTimeout         time.Duration
 }
 
 // NewIngressController returns a new ingress controller
 func NewIngressController(client federation_release_1_4.Interface) *IngressController {
+	glog.V(4).Infof("->NewIngressController V(4)")
 	broadcaster := record.NewBroadcaster()
 	broadcaster.StartRecordingToSink(eventsink.NewFederatedEventSink(client))
 	recorder := broadcaster.NewRecorder(api.EventSource{Component: "federated-ingress-controller"})
-
 	ic := &IngressController{
 		federatedApiClient:    client,
-		ingressReviewDelay:    time.Second * 10,
+		ingressReviewDelay:    time.Minute * 3,
```

Comment:
In theory they're only a backup in case there are bugs in the watch code.  But re-reviewing my code, those constants are used as retries on failure in a few places.  I'll put them back to their original values for now, until I've replaced those retry loops with exponential backoffs.